apps/services/schema.py

```diff
@@ -0,0 +1,23 @@
+import graphene 
+from graphene_django import DjangoObjectType
+from .models import Services, HairStyle
+
+
+class ServiceModel(DjangoObjectType):
+    class Meta:
+        model = Services
+
+class HairStyleModel(DjangoObjectType):
+    class Meta:
+        model = HairStyle
+
+class Query(graphene.ObjectType):
+    services = graphene.List(ServiceModel)
+    hairstyle = graphene.List(HairStyleModel)
+
+    def resolve_services(self, info, **kwargs):
+        return Services.objects.all()
+
+    def resolve_hairstyle(self, info, **kwargs):
```

Comment:
@KicaRonaldOkello, could you add a resolve for one hairstyle and service

Reply:
That will be added in another query. Where you make a query with an id, and that hairstyle is returned.